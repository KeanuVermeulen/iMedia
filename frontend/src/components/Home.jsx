// Task 19: Capstone Project - Compulsory Task 1 - Frontend - Components - Home.jsx

// --- Imports ---

import React from 'react';

// --- Rendered in App ---

class Home extends React.Component {
    render(){
        return(
            <div>
            	<h2>Welcome!</h2>
            </div>
        )
    }
}


export default Home