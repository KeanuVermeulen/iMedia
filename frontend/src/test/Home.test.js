// Task 19: Capstone Project - Compulsory Task 1 - Home.test.js

// This is a snapshot test for the 'Home.jsx' component of my app.

import React from 'react'; 
import renderer from 'react-test-renderer'; 
import Home from '../components/Home';

test('renders correctly', () => {
const tree = renderer
.create(<Home/>)
.toJSON();
expect(tree).toMatchSnapshot();
});