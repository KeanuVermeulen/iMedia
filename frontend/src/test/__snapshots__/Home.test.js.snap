// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Welcome!
  </h2>
</div>
`;
